cfme/infrastructure/config_management.py

```diff
@@ -364,15 +296,10 @@ def __init__(self, name, manager):
         self.name = name
         self.manager = manager
 
-    def navigate(self):
-        """Navigates to the profile's detail page"""
-        sel.force_navigate('infrastructure_config_manager_config_profile',
-            context={'manager': self.manager, 'profile': self})
-
     @property
     def systems(self):
         """Returns 'ConfigSystem' objects that are active under this profile"""
-        self.navigate()
+        navigate_to(self, 'Description')
```

Comment:
I also thought about this. I've put Description because the appeared view didn't look like usual Details view.
But I'd side with you that Details is more convenient as we used it everywhere.